src/frontend/styles/vars.pcss

```diff
@@ -1,42 +1,57 @@
 :root {
   --color-text-main: #313649;
   --color-text-second: #5d6068;
-  --color-line-gray: #E8E8EB;
+  --color-line-gray: #e8e8eb;
   --color-link-active: #2071cc;
-  --color-link-hover: #F3F6F8;
+  --color-link-hover: #f3f6f8;
   --color-bg-light: #f8f7fa;
   --color-page-active: #ff1767;
 
-  --color-button-primary: #3389FF;
-  --color-button-primary-hover: #2E7AE6;
-  --color-button-primary-active: #296DCC;
+  --color-button-primary: #3389ff;
+  --color-button-primary-hover: #2e7ae6;
+  --color-button-primary-active: #296dcc;
 
   --color-button-secondary: #717682;
-  --color-button-secondary-hover: #5D6068;
-  --color-button-secondary-active: #4B4F5B;
-
-  --color-button-warning: #EF5C5C;
-  --color-button-warning-hover: #D65151;
-  --color-button-warning-active: #BD4848;
+  --color-button-secondary-hover: #5d6068;
+  --color-button-secondary-active: #4b4f5b;
 
+  --color-button-warning: #ef5c5c;
+  --color-button-warning-hover: #d65151;
+  --color-button-warning-active: #bd4848;
 
   /**
    * Site layout sizes
    */
   --layout-padding-horizontal: 16px;
   --layout-padding-vertical: 30px;
+
   --layout-width-aside: 300px;
   --layout-width-main-col: 650px;
 
+  --layout-width-aside-right-title: 277px;
+  --layout-right-aside-resposive-width: 160px;
+  --layout-width-right-aside-available: calc(
+    50% - var(--layout-width-main-col) / 2 -
+      var(--layout-right-aside-resposive-width)
+  );
+  --layout-width-right-aside: max(
+    320px,
+    var(--layout-width-right-aside-available)
+  );
+  --layout-right-aside-padding-left: 30px;
```

Comment:
we have the `--layout-padding-horizontal` for that

Reply:
done